### DISCUSSION TITLE
Compare 2 date in the same table?

### DISCUSSION THREAD

**Discussion by hibangun** [mikro-orm/mikro-orm#2820] [Q&A]
Does anyone know how to compare 2 date in the same table?
In mongodb we can use something like this 
```db.T.find( { $where: function() { return this.date1 > this.date2 } } );```

How can we do in mikro orm?
Im using mikro-orm-postgre

**Comment by B4nan** [comment]
https://mikro-orm.io/docs/entity-manager#using-custom-sql-fragments

**Reply by hibangun** [reply]
can you gimme example?
in the example, its just lowercasing email
```const users = await em.find(User, { [expr('lower(email)')]: 'foo@bar.baz' }, {```

i try this way but seems doesnt work
```const users = await em.find(User, { [expr('created_at')]: expr('updated_at') }, {```
or this also doesnt work
```const users = await em.find(User, { updated_at: { $gte: expr('created_at') } }, {```
